dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityServiceTest.java

```diff
@@ -1017,7 +1015,7 @@ void shouldReturnTrackedEntityRelationshipsWithTei2Event()
         () -> assertEquals(eventA.getUid(), actual.getTo().getEvent().getUid()));
   }
 
-  private static List<String> uids(Collection<? extends BaseIdentifiableObject> trackedEntities) {
+  private List<String> uids(Collection<? extends BaseIdentifiableObject> trackedEntities) {
```

Comment:
Maybe we should rename this to something like

```suggestion
  private List<String> uids(Collection<? extends BaseIdentifiableObject> identifiableObjects) {
```

as its generic and not trackedEntities specific